docs/data/joy/main-features/color-inversion/ColorInversionFooter.tsx

```diff
@@ -28,14 +27,14 @@ export default function ColorInversionFooter() {
       color={color}
       invertedColors
       sx={{
-        ...(color !== 'warning' && {
+        ...(color !== 'neutral' && {
```

Comment:
Why has this changed?

<img width="820" alt="image" src="https://github.com/mui/material-ui/assets/18292247/38114cdd-9d3c-4725-a04f-15db8bca912e">

Reply:
No big reason, just so we started off with a less uncommon color. 
But that yellow is not looking great 😅 Any clues on making it better?